common/vm_wait_guest_fullname.yml

```diff
@@ -23,16 +23,24 @@
     datacenter: "{{ vsphere_host_datacenter }}"
     folder: "{{ vm_folder }}"
     name: "{{ vm_name }}"
+    schema: "vsphere"
+    properties:
+      - 'guest.guestId'
```

Comment:
I think these 3 other properties can be retrieved in `vm_get_guest_info.yml` using `vm_get_config.yml`, since this task is called "vm_wait_guest_fullname", it's only used for checking guestFullname is reported.

Reply:
Here only needs to execute vm_wait_guest_full_name for looking for these properties. If adding vm_get_config.yml to retrieve guest id and guest family, it means we need to execute vmware_guest_info module one more time. Just for saving time. Or I can just retrieve 'guest' property which include all of these properties, which may not be confusing. Your idea?